idlpy/src/types.c

```diff
@@ -146,13 +179,21 @@ emit_field(
 
     char *type = typename(ctx, type_spec);
 
+    //printf("emit_field name : %s , type %s \n", name , type);
+
     if (idl_is_default_case(parent)) {
+
+        //printf("emit_field idl_is_default_case name : %s , type %s \n", name , type);
+
         char *ctype;
         idl_asprintf(&ctype, "types.default[%s]", type);
         free(type);
         type = ctype;
     }
     else if (idl_is_case(parent)) {
+
+        //printf("emit_field idl_is_case name : %s , type %s \n", name , type);
```

Comment:
Please don't leave commented code in the PR.

Reply:
all the printfs that we have added have been deleted